### DISCUSSION TITLE
Language servers aren't working.

### DISCUSSION THREAD

**Discussion by kphanipavan** [coder/code-server#2445] [Q&A]
Hello,

I have started using code-server a week ago and it is very cool. It is now my editor of choise. But I have some issues it.

Language servers like PyLance, Jedi and Microsoft language servers for Python are not working at all.
Pylance had to be installed using a .vsix.

Does these work in code-server?

**Comment by oxy** [comment]
Hi!

You're technically not supposed to use Pylance with anything other than Microsoft's official Visual Studio Code build; see [their License](https://marketplace.visualstudio.com/items/ms-python.vscode-pylance/license) for the formal language. For licensing reasons, we can't ship Pylance for `code-server`.

MS Python not working from the extension store is a known issue - see https://github.com/cdr/code-server/issues/2341 - and we're working on fixing it!
